LoopKit/GlucoseRangeSchedule.swift

```diff
@@ -199,3 +199,18 @@ extension ClosedRange where Bound == HKQuantity {
         return DoubleRange(minValue: lowerBound.doubleValue(for: unit), maxValue: upperBound.doubleValue(for: unit))
     }
 }
+
+public extension DoubleRange {
+    init(_ val: ClosedRange<Double>) {
+        self.init(minValue: val.lowerBound, maxValue: val.upperBound)
+    }
+}
+
+public extension Optional where Wrapped == GlucoseRangeSchedule {
+    func scheduleRange() -> ClosedRange<HKQuantity> {
```

Comment:
I'm a bit confused about what this is doing - is it unwrapping the `scheduleRange`?

Reply:
It is allowing us to call `scheduleRange()` on an `Optional<GlucoseRangeSchedule>`, where if `nil` it would return an empty range.

Turns out I don't need that code anymore, so I'll remove it.